### PR TITLE
#159987395 Setup test with mocha and chai

### PR DESCRIPTION
**Pull request by DanielAdek**
#### What does this PR do?
Setup unit test for with mocha, chai, and chai-http for the project.

#### Description of Task to be completed?

- Install babel
-  Install mocha
-  Install chai
-  Install chai-HTTP
-  Initialize babel with `.babelrc`
-  create `app.js` for a simple app
-  create a simple test functionality

#### How should this be manually tested?

1. Clone repository
2. On your terminal run this command `git checkout ch-setup-test-159987395`
3. run `npm install` to install packages
4. run `npm test` to run test

#### Any background context you want to provide?
This set up a simple test environment for a comprehensive app

#### What are the relevant pivotal tracker stories?
#159987395

### CODE REVIEW

**Comment by johngorithm** [comment]
looks **good**

**Review by johngorithm — COMMENTED** [review]
looks good

**Comment by iAmao** [comment]
@DanielAdek Please fix all these HoundCI comments

File: test/test.spec.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            chai
+                .request(app)  
+                .get('/')
+                .end((err, res) => {
+                    res.body.should.be.an('object');
+                    res.body.should.have.property('message');
+                    res.body.message.should.eql('Welcome to the sims program');
+                    done();
+                });
+        })
```
[the remark this answers is not in the document]
Comment (iAmao) [reply test/test.spec.js]: @DanielAdek missing semi colon

File: test/test.spec.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    })
```
[the remark this answers is not in the document]
Comment (iAmao) [reply test/test.spec.js]: @DanielAdek Missing semicolon.

**Review by IyiKuyoro — APPROVED** [review]
Good and ready for merge.

**Review by iAmao — CHANGES_REQUESTED** [review]

**Review by iAmao — APPROVED** [review]

**Pull request merged**